platform/ui/src/components/ToolbarButton/ToolbarButton.tsx

```diff
@@ -31,58 +28,78 @@ const ToolbarButton = ({
   );
 
   const sizeToUse = size ?? 'toolbar';
-  const toolTipClassNameToUse =
-    toolTipClassName !== undefined
-      ? toolTipClassName
-      : sizeToUse === 'toolbar'
-        ? 'w-[40px] h-[40px]'
-        : 'w-[32px] h-[32px]';
 
-  return (
-    <div key={id}>
-      <Tooltip
```

Comment:
the logic of showing a dropdown content was used by the grid layout tool, now the grid layout tool is behaving in-correctly, it looks like if you hover on it the content will show as a tooltip, but clicking on it does nothing anymore.

We need it to have a tooltip like all other buttons, but clicking on it still brings the dropdown content (layout selector)